pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+image. When comparing both photos of the pirated and non-pirated drop off ports, it 
+seems there are no significant changes. Looking a little more closely, one can see
+that the pirated ships are spread out more widely in the region and not largely focused 
+like the red dots. The larger dots represent the level of drop off. The larger the dots,
+the more activity took place in the port.	
+
+---
+
+![image]({{ site.baseurl}}/assets/images/5.%20Distibution%20amount%20.png)
+
+The most significant change in the region is where the pirated ships dropped off 
+their slaves for sale compared to non-pirated vessels. This image shows there was 
+a market for pirated slave ships in the West Indies. The standard slave trade ships 
+are equally distributed in the slave markets. The larger the dots, the more vessels 
+ported and dropped off their slaves. The average slave trade led the ships to Rio in 
+Brazil and La Habana, Cuba. In pirated ships, the data shows the ports were more 
+concentrated and spread out.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/Captive%20by%20Pirates.png)
+
+This image is a  more clear representation of the ships and the results that followed
+once Pirates and Privateers took control of the vessels. The picture shows the changes 
+did not occur until the ship reached the region and ported in a different location than 
+non-pirated ships.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/Average%20Price%20of%20Slaves%20Over%20Time.jpg)
+![image]({{ site.baseurl}}/assets/images/Pirate%20Slave%20activity%20.png)
+
+These two images compare the price of slaves between 1804-1861 and the
+overall sale of pirated ships from 1700-1827. The standard tread is that, 
+by year, there was no major significant increase to sold pirated ships when 
+comparing the drop and rise on the cost of slave worth. The image shows pirates
+were not investing considerable energy to gather wealth by the slave trade. 
+Moreover, the representation does indicate the random interaction between pirates
+and slave ships. The number of slaves captured and sold by Pirates is deficient 
+compared to the significant traffic of slave ships in the area.
 
-This is where you where you discuss any (initial) conclusions you have come to. This is also a natural place to put any and all visualizations that you come up with. In fact, this is where you tell the reader what it is you discovered. You can use the first person if that helps you discuss the process. This is where you both describe to the reader what they are looking at (if they are looking at a visualization) and, most importantly, what your interpretation of the results is.
+---
 
-If you want to add an image you can do it like this....
+![image]({{ site.baseurl}}/assets/images/Pirates%20and%20actions%20with%20slaves.png)
 
-![Example Image]({{ site.baseurl }}/assets/images/example-image.png)
+The Relative Density comparing all pirated ships and its cargo shows that there
```

Comment:
You really went all out using multiple visualization tools, I love it. You tried to come at the data as many ways as possible.

I bet if you used Orange3 more, you might have found some really interesting things.